### PR TITLE
DOCS-8368 adding additional information on policy properties. Rewriti…

### PR DESCRIPTION
**Pull request by sleslie23**
DOCS-8368 adding additional information on policy properties. Rewriting intro paragraphs for SEO optimization.

### CODE REVIEW

File: modules/ROOT/pages/cap-policy.adoc [hunk 1]
```diff
@@ -3,39 +3,42 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The HTTP Limits policy prevents an attacker from sending large messages that consume all your bandwidth. +
-This policy checks TCP protocol message sizes and headers. This policy does not check content.
+Anypoint Security uses the HTTP Limits policy to prevent an attacker from sending large messages that can consume bandwidth and result performance degradation or system outage. This policy applies to all the APIs behind your Edge inbound endpoint. You can apply API Gateway policies to each API to enforce other API-specific policies, such as throttling and JSON threat protection.
```
Comment (clies212) [inline modules/ROOT/pages/cap-policy.adoc:6]: This looks a bit dicey: "large messages that can consume bandwidth and result performance degradation or system outage". Maybe something like: "large messages that can consume bandwidth which can cause performance degradation or denial of service."

File: modules/ROOT/pages/cap-policy.adoc [hunk 2]
```diff
@@ -3,39 +3,42 @@ ifndef::env-site,env-github[]
...
 
-This policy applies to all the APIs behind your Edge inbound endpoint. You can apply API Gateway policies to each API to enforce other API specific policies, such as throttling and JSON threat protection.
+The HTTP Limits policy limts the size of TCP protocol messages, paths, headers, and footers. This policy does not perform checks on the content of a message.
```
Comment (clies212) [inline modules/ROOT/pages/cap-policy.adoc:8]: Spelling mistake: "limts". Should be "limits".

File: modules/ROOT/pages/cap-policy.adoc [hunk 3]
```diff
@@ -3,39 +3,42 @@ ifndef::env-site,env-github[]
...
-|*Maximum Message Size* a| The default maximum message size allowed is 104857600. If request body scanning is enabled, this value should be set no larger than required to prevent attackers from abusing request body checking and exhausting resources. +
-Modify the message size based on your application needs, and base the size limits on what all of your APIs can handle. 
-|*Maximum Path Length* | The default maximum path size allowed is 4096 bytes.
-|*Maximum Length Of a Single Header* | The default maximum header length allowed for is 16384 bytes.
-|*Maximum Length Of a Single Trailer* | The default maximum trailer length allowed is 16384 bytes.
-|*Maximum Number Of Headers and Trailers* | The default maximum number of headers and trailers allowed is 32 kB.
+|*Maximum Message Size* a| The default maximum message size allowed is 104857600 bytes. +
+Modify this value based on the requirements of your application by setting the maximum size to a value based on a size your APIs can handle. +
+If the request body scanning option is enabled, do not set this value to a value larger than required to prevent attackers from abusing request body checking and exhausting resources.
```
Comment (clies212) [inline modules/ROOT/pages/cap-policy.adoc:31]: Delete "If the request body scanning option is enabled,". This text makes sense in the context of the WAF policy but is confusing when it only applies to the Limits policy. The sentence seems OK without that leading prepositional phrase from a technical content point of view.

**Review by clies212 — CHANGES_REQUESTED** [review]

**Comment by IsaacEldridge** [comment]
Closing this PR against https://github.com/mulesoft/docs-anypoint-security/pull/62 which is an updated duplicate

**Pull request closed without merging**